sdks/python/apache_beam/transforms/periodicsequence.py

```diff
@@ -48,14 +48,27 @@ def initial_restriction(self, element):
   def create_tracker(self, restriction):
     return OffsetRestrictionTracker(restriction)
 
-  def restriction_size(self, unused_element, restriction):
-    return restriction.size()
+  def restriction_size(self, element, restriction):
+    return sequence_backlog_bytes(element, time.time(), restriction)
 
   # On drain, immediately stop emitting new elements
   def truncate(self, unused_element, unused_restriction):
     return None
 
 
+def sequence_backlog_bytes(element, now, offset_range):
```

Comment:
```suggestion
def _sequence_backlog_bytes(element, now, offset_range):
```

(Since it's seems to be an internal helper)

Reply:
Is it important for the return value to be precise (e.g. up to a byte), or these are all estimates?

Reply:
better if we match output bytes for autoscaling, added a comment